juriscraper/pacer/free_documents.py

```diff
@@ -103,26 +99,25 @@ def download_pdf(self, pacer_case_id, pacer_document_number):
 
         Note that this doesn't support attachments yet.
         """
+        timeout = (60, 300)
         url = make_doc1_url(self.court_id, pacer_document_number, True)
         data = {
             'caseid': pacer_case_id,
             'got_receipt': '1',
         }
+
         logger.info("GETting PDF at URL: %s with params: %s" % (url, data))
-        r = self.session.get(
-            url,
-            params=data,
-            headers={'User-Agent': 'Juriscraper'},
-            verify=False,
-            timeout=300,
-        )
+        r = self.session.get(url, params=data, timeout=timeout)
```

Comment:
This is one of my crusades: https://github.com/kennethreitz/requests/issues/3070